salt/states/file.py

```diff
@@ -2390,15 +2390,19 @@ def managed(name,
                     **kwargs
                 )
 
-                if salt.utils.platform.is_windows():
-                    ret = __salt__['file.check_perms'](
-                        path=name,
-                        ret=ret,
-                        owner=win_owner,
-                        grant_perms=win_perms,
-                        deny_perms=win_deny_perms,
-                        inheritance=win_inheritance,
-                        reset=win_perms_reset)
+                try:
```

Comment:
This section of code would be cleaner if you encompassed just the `file.check_perms` command in the try/except instead of the `if is_windows` check as well.